packages/features/schedules/components/Schedule.tsx

```diff
@@ -193,8 +193,18 @@ export const DayRanges = <TFieldValues extends FieldValues>({
                 StartIcon={Plus}
                 onClick={() => {
                   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-                  const nextRange: any = getNextRange(getValues(`${name}.${fields.length - 1}`));
-                  if (nextRange) append(nextRange);
+                  const slotRange: any = getDateSlotRange(
+                    getValues(`${name}.${fields.length - 1}`),
+                    getValues(`${name}.0`)
+                  );
+
+                  if (slotRange?.append) {
+                    append(slotRange.append);
+                  }
+
+                  if (slotRange?.prepend) {
+                    prepend(slotRange.prepend);
```

Comment:
Once the slot end time will reach `11.59 pm` it will start prepending the slots meaning my first slot is 11.00 am -12.00 pm. I still have slot windows before 11.00 am so I will start prepending the slots. See loom video for more clarity